tensorboard/webapp/runs/views/runs_table/filterbar_test.ts

```diff
@@ -61,6 +66,17 @@ const fakeFilterMap = new Map<string, DiscreteFilter | IntervalFilter>([
   ['filter2', intervalFilter],
 ]);
 
+@Component({
+  selector: 'testable-component',
+  template: ` <filterbar></filterbar> `,
+})
+class TestableComponent {
+  constructor(
+    readonly customModal: CustomModal,
+    readonly vcRef: ViewContainerRef
```

Comment:
Is vcRef still needed?

Reply:
Good point, the inner component already has its own ViewContainerRef. Removed (same for below)